src/pages/EditRequestDescriptionPage.js

```diff
@@ -77,7 +77,7 @@ function EditRequestDescriptionPage({defaultDescription, onSubmit}) {
                         }}
                         autoGrowHeight
                         containerStyles={[styles.autoGrowHeightMultilineInput]}
-                        submitOnEnter={!Browser.isMobile()}
+                        submitOnEnter={!canUseTouchScreen()}
```

Comment:
Could we walk over what's actually happening here and why?

We want to make this effectively multiline text input be able to submit the form ("submit on Enter"), but only when we can't use the touch screen (think: desktop web). What is the behavior in such a case? How do we add a new line? Has the behavior changed?

Reply:
In web where we have keyboard, we generally use Shift + enter to enter a new line. But in phones, we can't use shift + enter as it is not available. So we don't submit here at all on enter.

Reply:
It this case-by-case logic? Do we ever want a multi-line input that you can't reasonably make multiline on mobile?

I'm wondering whether this shouldn't just be handled centrally, with a nice code comment rephrasing what you just said.

Reply:
You have raised a good point.
We can do that. But I am not sure we are are allowed to do that.

Let me summon @luacmartins as form manager to decide here.

Reply:
What exactly are you trying to do? I see `whether this shouldn't just be handled centrally`, but what does that mean?

Reply:
We have to decide between 2 options:
1. Developer will never be able to submit form through enter key on multiline input in mobile devices (both mWeb/native) by hardcoding the conditions inside FormProvider.
2. Let it left as it is on the developer choice (this is current implementation).

Reply:
Thanks for the input.